src/main/java/wolox/training/models/Book.java

```diff
@@ -102,6 +106,7 @@ public String getTitle() {
     }
 
     public void setTitle(String title) {
+        Preconditions.checkNotNull(title);
```

Comment:
The same as the previous comment.

Reply:
I put the messages and added it like constants.